### ISSUE TITLE
Error: Cannot read property 'promise' of null

### ISSUE THREAD

**Issue by newmuis** [ampproject/amphtml#24246]
```
Error: Cannot read property 'promise' of null
at (https://raw.githubusercontent.com/ampproject/amphtml/1908231648370/extensions/amp-video-iframe/0.1/amp-video-iframe.js:388)
at postMessage_ (https://raw.githubusercontent.com/ampproject/amphtml/1908231648370/extensions/amp-video-iframe/0.1/amp-video-iframe.js:375)
at (https://raw.githubusercontent.com/ampproject/amphtml/1908231648370/extensions/amp-video-iframe/0.1/amp-video-iframe.js:410)
at pause (https://raw.githubusercontent.com/ampproject/amphtml/1908231648370/extensions/amp-video-iframe/0.1/amp-video-iframe.js:405)
at pauseCallback (https://raw.githubusercontent.com/ampproject/amphtml/1908231648370/src/custom-element.js:1330)
at pauseCallback (https://raw.githubusercontent.com/ampproject/amphtml/1908231648370/src/service/resource.js:1053)
at pause (https://raw.githubusercontent.com/ampproject/amphtml/1908231648370/src/service/resource.js:1077)
at unload (https://raw.githubusercontent.com/ampproject/amphtml/1908231648370/src/service/resources-impl.js:2573)
at forEach (https://raw.githubusercontent.com/ampproject/amphtml/1908231648370/src/service/resources-impl.js:2572)
```

go/ampe/CJCcoczlpdyaQQ

**Comment by skyzanellap** [comment]
Hi, I have the same error. Can you help me to solve it? 
In console I noticed this error:
`
Uncaught (in promise) TypeError: Cannot read property 'promise' of null
    at nb (amp-video-iframe.js:389)
    at Z (amp-video-iframe.js:376)
    at W.push.f.g.pause (amp-video-iframe.js:411)
    at W.push.f.g.pauseCallback (amp-video-iframe.js:406)
    at HTMLElement.Ej.b.pauseCallback (custom-element.js:1311)
    at $i.f.pauseOnRemove (resource.js:977)
    at Ik (resources-impl.js:525)
    at resources-impl.js:498
`
Thanks!
